apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/Disclaimer/index.tsx

```diff
@@ -43,14 +43,16 @@ export default function Disclaimer({
 
       <Box mt={6} height="1px" width="100%" backgroundColor={"translucentGrey"} />
 
-      <Flex mt={6} flexDirection={"row"} alignItems={"center"}>
-        <Box mr={2}>
-          <Icons.Eye color={"smoke"} />
-        </Box>
-        <Text fontSize={14} color="smoke">
-          {t("web3hub.disclaimer.clearSigningEnabled")}
-        </Text>
-      </Flex>
+      {manifest?.categories.includes("clear signing") && (
```

Comment:
I would create a const like `clearSigningEnabled` with a memo as includes still iterate over an undefined number of elements and you don't want to do this every time in the render function